### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by clintpurser**
updating the readme, clarifying steps to use the package

### CODE REVIEW

**Review by kevin49999 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,9 +4,20 @@ A way to use the gonmea package in Flutter
 
 ## Getting Started
 
-This project is a starting point for a Flutter
-[FFI plugin](https://docs.flutter.dev/development/platform-integration/c-interop),
-a specialized package that includes native code directly invoked with Dart FFI.
+This package contains the Go implementation where it is using the gonmea package to parse nmea readings. To build an app that consumes this package, do the following:
```
Comment (Oabraham1) [inline README.md]: Do you think it'll useful to link the gonmea package on this line [gonmea](https://github.com/erh/gonmea)

File: README.md [hunk 2]
```diff
@@ -4,9 +4,20 @@ A way to use the gonmea package in Flutter
...
+
+* `cd src` then run `make ios` to generate the ios binaries for both simulator and physical iPhone devices. 
+
+* Run `make android` to generate binaries for different android architectures.
+
+* In the root directory of the project, run `flutter pub run ffigen --config ffigen.yaml` to auto-generate the dart bindings from the header file
+
+* All the implementation for the public facing API exists in the lib/flutter_nmea_impl.dart file.
+
+* There is an example app to help test the implementation. `cd example && flutter run` to run the example app
```
Comment (Oabraham1) [inline README.md:19]: Love this overall 👍🏾

File: README.md (hunk 1 above)
Comment (clintpurser) [reply README.md]: yes

**Review by Oabraham1 — APPROVED** [review]
lgtm

**Pull request merged**